### DISCUSSION TITLE
Blast from the past site designs

### DISCUSSION THREAD

**Discussion by amnuts** [amnuts/Amnuts#13] [General]
Today I found a few different versions of the old Amnuts site, so I thought I'd share these little throw-backs...

Question is; should I resurrect one for the amnuts.net domain, relive the glory days, as it were, go for something new, or just let amnuts.net stay a redirect to this repo?

## Version 1

This brings back some really fond memories for me.

### Main page
![amnuts www v1](https://user-images.githubusercontent.com/684421/219877133-48d82dfa-8501-4480-a656-584dc83efec1.png)

### Sub-page
![amnuts www v1 - inner](https://user-images.githubusercontent.com/684421/219877184-5390c5c7-9374-4b4b-886b-cf0c833201e9.png)

## Version 2

This is probably the most memorable one for me as it was around the longest.

### Main page
![amnuts www v2](https://user-images.githubusercontent.com/684421/219877248-eee7631f-c5af-4ae9-a0c6-491635ad840f.png)

### Sub-page
![amnuts www v2 - inner](https://user-images.githubusercontent.com/684421/219877255-dcab6d9d-ecb7-469f-9b2e-35a790ca3861.png)

## Version 3

This was never put into production.

### Main page
![amnuts www v3](https://user-images.githubusercontent.com/684421/219877280-c7a30832-1f80-4553-b6cb-0887a8b9c0a5.png)

## The development site

![amnuts dev site](https://user-images.githubusercontent.com/684421/219877292-b817bacb-a9d9-4b9a-b64b-776afafb8088.png)

**Comment by blindsight** [comment]
Version 2! Mostly bc of memories. Docker seems like the only safe way to run a talker these days.

**Comment by Uzume** [comment]
I recommend you create an orphan `gh-pages` branch and make a new minimal website for Amnuts. You will want to include a GH repo link and you can add the historic versions of the website by organizing them into some folders and linking to them there. This is called a GitHub Pages Project site.

You can make the orphan `gh-pages` branch via:
```
git checkout –orphan <branch name>
```
Then just delete any leftover files, create or copy in what you want, and add and check them in like normal.

Then you can look at it via: http://amnuts.github.io/Amnuts. There are ways to control how things get deployed (in the settings for each repo) but that should work by default.

Read more at:
- https://pages.github.com/
- https://docs.github.com/en/pages

Then you can redirect amnuts.net to http://amnuts.github.io/Amnuts or perhaps http://amnuts.github.io/ (if you figure out how and want to take over that too). Perhaps an even better option is to research how to host your site via GitHub Pages while avoiding the redirect that causes the URL to change in a user's browser: https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site

**Comment by krisprad25** [comment]
Brings back some memories mate!
